.github/workflows/velox_weekly.yml

```diff
@@ -99,3 +100,102 @@ jobs:
           sudo apt-get install -y openjdk-8-jdk
           export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
           cd $GITHUB_WORKSPACE/ && ./dev/package.sh
+
+  build-on-openeuler:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "openeuler/openeuler:24.03" ]
+    if: ${{ startsWith(github.repository, 'apache/') }}
+    runs-on: ubuntu-24.04-arm
+    container: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: build
+        run: |
+          dnf update -y && dnf install -y java-1.8.0-openjdk-devel sudo patch wget git perl
+          $SETUP install_maven
+          export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
+          export MAVEN_HOME=/usr/lib/maven
+          export PATH=${PATH}:${MAVEN_HOME}/bin:${JAVA_HOME}/bin
+          cd $GITHUB_WORKSPACE/ && ./dev/package.sh
```

Comment:
You can start a docker here to run a tpc-h test also. Please set `spark.gluten.loadLibFromJar=true` to make sure all required native libs packed in third-party jar are loaded at runtime.

The code path of loading lib from jar is not verified in CI. We are considering to add this test for other OS in subsequent pr.